docs/styles/_components.scss

```diff
@@ -6,6 +6,7 @@
 @import '../../components/01-globals/mixins/clearfix';
 @import '../../components/01-globals/mixins/grid-column-width';
 @import '../../components/01-globals/mixins/mq';
+@import '../../components/01-globals/mixins/svg-uri';
```

Comment:
I'm not convenience about this solution, but I'm not saying it's bad. I think with sprote for css background will be better b/c using a lot of svg inside the css we can increase amount of code and time of generate. But we can try this attitude :)

Reply:
as I said we can have control of color here. If you would like to show sprite solution for rating feel free as my try with svg for this case didn't convinced me (but also maybe I make sth wrong)

Reply:
I think we should try with sprite but we can do it later, I've create a separate issue: #277